lib/XMLHttpRequest.js

```diff
@@ -391,7 +452,61 @@ function XMLHttpRequest(opts) {
     }
 
     // Load files off the local filesystem (file://)
+    // or data from Data URI (data:)
     if (local) {
+      if (datauri) try {
+        // Triming from original url string since 
+        var data = url.href.substr(5, settings.url.length);
+
+        // separator between header and actual data
+        var parts = data.split(",", 2);
+
+        if (parts.length < 2) throw "Invalid URL";
+
+        var base64 = false;
+        // check if header part has base64
+        var data_headers = parts[0].split(";");
```

Comment:
Let's use camelCase everywhere for consistency

Reply:
alright i will change this one too